pkg/scheduler/api/numa_info.go

```diff
@@ -118,6 +141,50 @@ func (info *NumatopoInfo) Release(resSets ResNumaSets) {
 	}
 }
 
+func GetPodResourceNumaInfo(ti *TaskInfo) map[int]v1.ResourceList {
+	if ti.NumaInfo != nil && len(ti.NumaInfo.ResMap) > 0 {
+		return ti.NumaInfo.ResMap
+	}
+
+	if _, ok := ti.Pod.Annotations[topologyDecisionAnnotation]; !ok {
+		return nil
+	}
+
+	decision := PodResourceDecision{}
+	err := json.Unmarshal([]byte(ti.Pod.Annotations[topologyDecisionAnnotation]), &decision)
+	if err != nil {
+		return nil
+	}
+
+	return decision.NUMAResources
+}
+
+func (info *NumatopoInfo) AddTask(ti *TaskInfo) {
```

Comment:
please add the function description for all new added functions like `AddTask`, `RemoveTask`.

Reply:
done, added the  function description